suzieq/shared/poller_error_codes.py

```diff
@@ -0,0 +1,60 @@
+# List of normalized error codes used in the poller
+
+from enum import IntEnum
+
+# Ensure you never copy backwards compat values
+
+
+class SqPollerStatusCode(IntEnum):
+    '''SuzieQ Poller Status Codes and their strings'''
+
+    OK = 0, 'OK'
+    HTTP_OK = 200, 'OK'
+    PERMISSION_DENIED = 1001, 'Permission Denied'
+    UNAUTHORIZED = 1002, 'Unauthorized'
+    LOGIN_FAIL = 1003, 'Login Failed'
+    PRIV_ESC_FAIL = 1004, 'Privilege Escalation Failed'
+    NON_FINAL_FAILURES = 1005, 'Dummy to mark boundary, never set'
+    CMD_NOT_FOUND = 1006, 'Command Not Found'
```

Comment:
Can we use 2000+ for non final failures?

Reply:
Done